test/unit/gateways/realex_test.rb

```diff
@@ -51,6 +51,33 @@ def setup
     @amount = 100
   end
 
+  def test_initialize_sets_refund_and_credit_hashes
+    refund_secret = 'refund'
+    rebate_secret = 'rebate'
+
+    gateway = RealexGateway.new({
```

Comment:
The curly braces aren't needed inside the `Gateway.new` calls.

Reply:
Removed.

Reply:
Thanks!